types/webpack-bundle-analyzer/index.d.ts

```diff
@@ -8,6 +8,8 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.7
 
+/// <reference types="node" />
```

Comment:
Why?

Reply:
I'm not sure, again this was copied from #52044. See the comment thread for the same change here where at the time at least it was accepted as an acceptable solution: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52044/files#r604233809

Reply:
> Why?

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52044/files#r604233809

Reply:
Thanks!